hubblestack/daemon.py

```diff
@@ -162,6 +164,27 @@ def main():
             log.exception('Error executing schedule')
         time.sleep(__opts__.get('scheduler_sleep_frequency', 0.5))
 
+def getLastRunByBuckets(buckets, seconds):
```

Comment:
Could you make this function lowercase instead of camelCase? Python only uses camelCase on class names. You can add underscores if you think it improves readability but they are not necessary.